api/envoy/api/v3alpha/BUILD

```diff
@@ -7,7 +7,7 @@ licenses(["notice"])  # Apache 2
 api_proto_package(
     has_services = True,
     deps = [
-        "//envoy/api/v3alpha/auth:pkg",
+        "//envoy/api/v2/auth:pkg",
```

Comment:
@htuch I merged master and then try to use fix_format to address merge conflicts in v3alpha, then this sneaked in and passes format check. seems like a bug.

Reply:
Yeah, this seems a bit off. I'll take a look.